desci-server/src/controllers/auth/magic.ts

```diff
@@ -46,7 +46,7 @@ export const magic = async (req: Request, res: Response, next: NextFunction) =>
         res.cookie('auth', token, {
           maxAge: oneDay,
           httpOnly: true,
-          // secure: process.env.NODE_ENV === 'production', // unsafe, but only used for local dev
+          secure: false, // unsafe, but only used for local dev
```

Comment:
I'd personally use the previous expression in case this config is forgotten and we end up launching it publicly anyway :)